guides/NEW_APP.md

```diff
@@ -0,0 +1,50 @@
+Creating a new app
+==================
+
+The following steps will guide you through add adding a new app to our system that is built by the CI system and deployed to the live environment.
+
+Prerequisites
+-------------
+
+In order to follow this guide you will need:
+
+* A github account with access to the ONSdigital organisation
+
+* A login for the concourse CI environment
+
+* [dp-cli](https://github.com/ONSdigital/dp-cli)
+
+* Git:
+
+  ```
+  brew install git
+  ```
+
+* Terraform >= 0.12:
+
+  ```
+  brew install terraform
+  ```
+
+* Fly cli (can be downloaded from the links in the corner of the concourse UI)
+
+Getting started
+---------------
+
+These steps will create a new app that is built by CI and deployed to the develop and production environments.  If any of the following github links 404 it is likely that you need to be added to the ONSdigital organisation.
+
+1. Determine what language you want to use and technology your app will likely use
+
+2. Pick a local dev port for your app and add it to the [local ports list](PORTS.md)
+
+3. Use the [`dp-cli`](https://github.com/ONSdigital/dp-cli) to generate the new repo and populate it with the standard boilerplate files:
+
+   ```
+   dp-utils generate-project
+   ```
+
+4. [Add the new app to the environment](https://github.com/ONSdigital/dp-setup#adding-a-new-app)
+
+5. [Add the new app to CI](https://github.com/ONSdigital/dp-ci#adding-a-new-app-to-ci)
+
+6. Check that your new app has successfully deployed to develop
```

Comment:
Considering the level of detail this guide goes to, it might be worth adding how to check that the app has been deployed (I imagine check concourse) rather than just stating check it.